packages/dev/docs/pages/releases/2023-10-06.mdx

```diff
@@ -0,0 +1,102 @@
+
+date: 2023-10-06
+---
+
+# October 06, 2023 Release
+
+In this release, we've extended support for links across all collection components! To streamline integration with client-side routers, we've introduced the `RouterProvider` component which will allow you to seamlessly incorproate client-site routers without the need to wrap a router Link component ([see docs]()). We have also continued to make improvements and bug fixes to [React Aria Components](../react-aria/react-aria-components.html) since its beta release. Internal contexts have been refactored to allow for greater customizability ([see docs](../react-aria/advanced.html)), MenuTrigger and DialogTrigger now use PressResponder to work with any pressable component, and updated documentation to help improve adoption. React Spectrum also has some new additons, including [DropZone](../react-spectrum/DropZone.html) and the [isPending](../react-spectrum/Button.html#pending) prop to Button. 
+
+As always, huge thanks to everyone in our community for their feedback and contributions to make this release possible!
+
+### FileTrigger API update
+
+This release of FileTrigger includes an API change. If you were using a previous version, you will need to update your code. See below for details.
+- The `OnChange` prop has been replaced with `OnSelect`.
+
+## Enhancements
+
+- Add staticColor prop to ProgressCircle and ProgressBar - [@iamwillpowell](https://github.com/iamwillpowell) - [PR](https://github.com/adobe/react-spectrum/pull/5108)
+- Add support for 'none' as a BorderSizeValue to hide border - [@pr7prashant](https://github.com/pr7prashant) - [PR](https://github.com/adobe/react-spectrum/pull/5012)
+- Support links in collection components - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/4993)
+- Cover all plural forms in NumberParser - [@jbovenschen](https://github.com/jbovenschen) - [PR](https://github.com/adobe/react-spectrum/pull/5134)
+- Add TypeScript StrictMode to React Stately utils - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/3926)
+
+## Fixes
+
+- Update Toast placement to bottom center - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5107)
+- Fix Unavailable MenuItem props - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/5153)
+- Fix Popover placement for placement="start"|"end" - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/5137)
+- Remove Express icon dependency - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/5130)
```

Comment:
no need to include this, it was part of Update Express Breadcrumb icon, just turned into two PRs instead of 1 because we decided to alter the implementation